en/core-api/e-wallet.md

```diff
@@ -88,19 +100,36 @@ curl -X POST \
 
 ```
 
+
+
+<details>
+<summary><b>Post Body</b></summary>
+<article>
+
+| Requirement         | Description                                                  | Type   | Required |
+| ------------------- | ------------------------------------------------------------ | ------ | -------- |
+| transaction_details | Details of the transaction such as order_id and gross_amount. | -      | Required |
+| order_id            | Transaction order ID, defined from your side.                | String | Required |
+| gross_amount        | Total amount of transaction, defined from your side.         | String | Required |
+| payment_type        | Set to `gopay`.                                              | String | Required |
+
+</article>
+</details>
+
```

Comment:
This `Post Body` section is relevant to all languages, so please place it outside the tabbed language section. Where it always visible no matter what languages is being used.